### ISSUE TITLE
clean install gives SQL error

### ISSUE THREAD

**Issue by PanMan** [wyveo/craftcms-docker#5]
I just installed this, but get an SQL error from postgress:
```
postgres_1   | ERROR:  relation "sites" does not exist at character 70
postgres_1   | STATEMENT:  SELECT "id", "name", "handle", "language", "hasUrls", "baseUrl"
postgres_1   | 	FROM "sites"
postgres_1   | 	ORDER BY "sortOrder"
```
any hints?

**Comment by colinwilson** [comment]
Looks like the install is querying the DB before any tables or columns exist (see https://github.com/craftcms/cms/issues/1333).

The installation should still complete without issue and you shouldn't see this error again anytime after.

Will raise this with the craft team, see if they have any suggestions.

**Comment by colinwilson** [comment]
Resolved by 1f24b56a2e1208d23eb7eef629077580d15a051d
